packages/techdocs-container/techdocs-core/src/core.py

```diff
@@ -30,6 +30,10 @@ def on_config(self, config):
             '{\n  "site_name": "{{ config.site_name }}",\n  "site_description": "{{ config.site_description }}"\n}'
         )
 
+        mdx_configs_override = {}
+        if "mdx_configs" in config.keys():
+            mdx_configs_override = config["mdx_configs"].copy()
+
```

Comment:
This too, I guess. :)

```suggestion
        mdx_configs_override = {}
        if "mdx_configs" in config:
            mdx_configs_override = config["mdx_configs"].copy()

```